go.mod

```diff
@@ -18,43 +20,48 @@ require (
 	github.com/prometheus/common v0.14.0
 	github.com/shurcooL/httpfs v0.0.0-20190707220628-8d4bc4ba7749 // indirect
 	github.com/shurcooL/vfsgen v0.0.0-20181202132449-6a9ea43bcacd
-	go.opencensus.io v0.22.2
-	golang.org/x/net v0.0.0-20200707034311-ab3426394381
-	golang.org/x/sys v0.0.0-20201015000850-e3ed0017c211
-	golang.org/x/time v0.0.0-20191024005414-555d28b269f0
-	google.golang.org/grpc v1.27.0
-	k8s.io/api v0.19.4
-	k8s.io/apimachinery v0.19.4
-	k8s.io/client-go v0.19.4
-	k8s.io/cri-api v0.0.0
-	k8s.io/klog/v2 v2.4.0
-	k8s.io/kubernetes v1.19.4
+	go.opencensus.io v0.22.3
+	golang.org/x/net v0.0.0-20210226172049-e18ecbb05110
+	golang.org/x/sys v0.0.0-20210511113859-b0526f3d8744
+	golang.org/x/time v0.0.0-20210220033141-f8bda1e9f3ba
+	google.golang.org/grpc v1.33.2
+	k8s.io/api v0.21.0
+	k8s.io/apimachinery v0.21.0
+	k8s.io/client-go v0.21.0
+	k8s.io/cri-api v0.21.0
+	k8s.io/klog/v2 v2.8.0
+	k8s.io/kubernetes v1.21.0
 	sigs.k8s.io/yaml v1.2.0
 )
 
 replace (
+	github.com/containerd/nri => github.com/klihub/nri v0.0.0-20210916151032-5665bdaef70c
```

Comment:
NRI replace shouldn't probably be in this commit.

Reply:
Yeah, otherwise lgtm

Reply:
Thanks fixed that one.

Now it looks to me like `apimachinery v0.22.1` (`goresctrl` commit [`6d75e171`](https://github.com/intel/goresctrl/commit/6d75e171db261ba93da99061108bafb2282c43c2)) causes an inherently incompilable k8s dependency set to get pulled in. I could not find any other fix than downgrade that dependency to not later than `v0.21.0`. AFAICT is the latest tagged release for which all the necessary k8s subpackages split out from `staging` have been properly updated. A real pain in the butt...

Here is a [pending PR](https://github.com/intel/goresctrl/pull/78) doing just that.